### ISSUE TITLE
Can't touch NPC

### ISSUE THREAD

**Issue by eBeeT** [CitizensDev/Citizens#898]
Hello, I noticed that after upgrading from 1.1 to 1.2 minecraft version I can't touch citizens.
I upgraded citizens, but after that my old NCP doesn't work.

NPC Works only when I create a new NPC.
But old NPC doesn't work.

**Comment by ghost** [comment]
I second this.

**Comment by ghost** [comment]
/reload fixes the problem, after it loads.

**Comment by eBeeT** [comment]
It doesn't help me.

**Comment by Sydael** [comment]
I noticed that when a player is online while npc is created, the player can interact with it. And it applies to the creator and any other person that is online on server while creation. But when that player logs out or isn't on while NPC is created, he cannot interact with it. Also the creator of NPC, once relogged after creating, he cannot interact with it either.
Reloading works, but only for people that are online. And even so, when they relog, the problem reappears.

**Comment by damopewpew** [comment]
I made this comment on a similar bug higher up. Interaction seems to work when you walk away from the npc and then come back, but teleporting to it is when the bug happens and you can't interact.

Trading does not work though and you can take things out of the inventory without it making a purchase

**Comment by cascaid** [comment]
I am having this problem too, thanks for the /reload tip, it fixes it for the short term...

**Comment by eBeeT** [comment]
Yes but only for short time. After logout npc doesn't work. PLEASE FIX.
